### PR TITLE
[Carthage] define PIN_REMOTE_IMAGE in .pch for non-CocoaPod users

### PR DESCRIPTION
**Pull request by hannahmbanana**
Resolves Issue 1 of #1780. 

Users who don't use CocoaPods will have PIN_REMOTE_IMAGE defined in .pch file.

### CODE REVIEW

File: AsyncDisplayKit/Details/ASPINRemoteImageDownloader.m [hunk 1]
```diff
@@ -85,7 +85,23 @@ - (PINRemoteImageManager *)sharedPINRemoteImageManager
   static PINRemoteImageManager *sharedPINRemoteImageManager = nil;
   static dispatch_once_t onceToken;
   dispatch_once(&onceToken, ^{
+  
 #if PIN_ANIMATED_AVAILABLE
+    // Check that Carthage users have linked both PINRemoteImage & PINCache by testing for one file each
+    if (!(NSClassFromString(@"PINRemoteImageManager.m"))) {
```
Comment (maicki) [inline AsyncDisplayKit/Details/ASPINRemoteImageDownloader.m]: You should not to add an `.m` in this case ... just use `NSClassFromStirng(@"PINRemoteImageManader")`

File: AsyncDisplayKit/Details/ASPINRemoteImageDownloader.m [hunk 2]
```diff
@@ -85,7 +85,23 @@ - (PINRemoteImageManager *)sharedPINRemoteImageManager
...
+        NSException *e = [NSException
+                          exceptionWithName:@"FrameworkSetupException"
+                          reason:@"Missing the path to the PINRemoteImage framework."
+                          userInfo:nil];
+        @throw e;
+    }
+    if (!(NSClassFromString(@"PINCache.m"))) {
```
Comment (maicki) [inline AsyncDisplayKit/Details/ASPINRemoteImageDownloader.m]: I guess same for this case as above, you should remove the `.m` at the end of the class

**Comment by hannahmbanana** [comment]
Thanks @nickvelloff for testing this out!

**Comment by nickvelloff** [comment]
@hannahmbanana You bet! Thanks for the solution!

**Pull request merged**